mappings/net/minecraft/command/arguments/MobEffectArgumentType.mapping

```diff
@@ -2,7 +2,7 @@ CLASS cq net/minecraft/command/arguments/MobEffectArgumentType
 	FIELD a INVALID_EFFECT_EXCEPTION Lcom/mojang/brigadier/exceptions/DynamicCommandExceptionType;
 	FIELD b EXAMPLES Ljava/util/Collection;
 	METHOD a create ()Lcq;
-	METHOD a getEffectArgument (Lcom/mojang/brigadier/context/CommandContext;Ljava/lang/String;)Lahv;
+	METHOD a getMobEffect (Lcom/mojang/brigadier/context/CommandContext;Ljava/lang/String;)Lahv;
```

Comment:
Don't we call these StatusEffects, or am I missing something?

Reply:
Yes, I'll rename this. It was named based on the strings in that class.